ara/models.py

```diff
@@ -251,6 +251,7 @@ class Host(db.Model):
     id = std_pkey()
     name = db.Column(db.String(255), unique=True, index=True)
 
+    facts = one_to_many('HostFacts', backref='host')
```

Comment:
This should be a one-to-one relationship (we're not trying to maintain a *history* of facts for a host).  That's going to look something like:

    one_to_one = functools.partial(
        db.relationship, passive_deletes=False,
        cascade='all, delete-orphan', uselist=False)

And then:

    facts = one_to_one('HostFacts', backref='host')